pkg/util/iptables/testing/fake.go

```diff
@@ -26,80 +26,106 @@ import (
 )
 
 const (
+	// Destination represents the destination address flag
 	Destination = "-d "
-	Source      = "-s "
-	DPort       = "--dport "
-	Protocol    = "-p "
-	Jump        = "-j "
-	Reject      = "REJECT"
-	ToDest      = "--to-destination "
-	Recent      = "recent "
-	MatchSet    = "--match-set "
-	SrcType     = "--src-type "
-	Masquerade  = "MASQUERADE "
+	// Source represents the source address flag
+	Source = "-s "
+	// DPort represents the destination port
+	DPort = "--dport "
```

Comment:
make sure all of these have "`flag`" at the end. (`DPort` isn't the destination port, it's the destination port *flag*.)

Reply:
gotcha, added it.